### DISCUSSION TITLE
No feedback when I run this code

### DISCUSSION THREAD

**Discussion by Wangyao1997-sdu** [ocornut/imgui#5831] [New Users Build/Link/Run issues ONLY!]
I want start to use imgui with this code, but there is no feedback when I run it.
`import imgui
imgui.create_context()
imgui.get_io().display_size = 100, 100
imgui.get_io().fonts.get_tex_data_as_rgba32()
imgui.new_frame()
imgui.begin("Your first window!", True)
imgui.text("Hello world!")
imgui.end()
imgui.render()
imgui.end_frame()`

Python 3.8
MacOS 13.0
imgui version == 1.4.1

**Comment by ocornut** [comment]
It looks like you are using a custom backend/framework so please ask whereever you got that backend/framework.

Dear ImGui 1.41 is dated June 2015 so I guess your version number "1.4.1" is not an actual Dear ImGui version.
